packages/esm-appointments-app/src/home-appointments/appointments-list.component.tsx

```diff
@@ -26,6 +26,7 @@ import {
   usePagination,
   useSession,
   userHasAccess,
+  ConfigObject,
 } from '@openmrs/esm-framework';
 import { ActionsMenu } from './appointment-actions.component';
 import { EmptyDataIllustration } from './empty-data-illustration.component';
```

Comment:
@CynthiaKamau  do you happen to know where this appointments-list component is used, so we can test our changes properly?  Thanks!